### PR TITLE
bandanas now are properly shown

### PR DESCRIPTION
**Pull request by SrReptile**
bandana time.

## About The Pull Request

little changes
## Why It's Good For The Game

they no longer look ugly af

## Changelog
:cl:SrReptile
code: Blockhair added in the bandanas
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -76,6 +76,7 @@
 /obj/item/clothing/head/bandana/green
 	name = "green bandana"
 	desc = "It's a green bandana with some fine nanotech lining."
+	flags_inv = BLOCKHAIR
```
Comment (TorinoFermic) [inline code/modules/clothing/head/misc.dm:79]: No need for that, /obj/item/clothing/head/bandana is the parent of this object. It already inherits it.

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -239,6 +241,7 @@
 /obj/item/clothing/head/bandana/orange //themij: Taryn Kifer
 	name = "orange bandana"
 	desc = "An orange piece of cloth, worn on the head."
+	flags_inv = BLOCKHAIR
```
Comment (TorinoFermic) [inline code/modules/clothing/head/misc.dm:244]: Ditto No need for that, /obj/item/clothing/head/bandana is the parent of this object. It already inherits it.

**Comment by GrayRachnid** [comment]
Are you sure this fixes them? Can you post ingame screenshot? As far as I was aware the sprites were missing or something.

**Comment by SrReptile** [comment]
test

File: code/modules/clothing/head/misc.dm (hunk 1 above)
Comment (SrReptile) [reply code/modules/clothing/head/misc.dm:79]: If  I erase them, the hair doesn't dissapear in the others bandanas

File: code/modules/clothing/head/misc.dm (hunk 2 above)
Comment (SrReptile) [reply code/modules/clothing/head/misc.dm:244]: If I erase them, the hair doesn't dissapear in the others bandanas

File: code/modules/clothing/head/misc.dm (hunk 1 above)
Comment (TorinoFermic) [reply code/modules/clothing/head/misc.dm:79]: It won't, you just assigned something in flags_inv in /obj/item/clothing/head/bandana and it gets shared with childs as long they don't get modified again.

**Pull request merged**